homepairs/homepairsUI/src/screens/Auth/SignUpScreen/SignUpScreenBase.tsx

```diff
@@ -335,7 +335,7 @@ export class SignUpScreenBase extends React.Component<SignUpProps,SignUpState> {
                     alignItems: 'center',
                     marginTop: 25,
                     marginBottom: 20,
-                    width: 400,
+                    width: Platform.OS === 'web'? 400 : width*0.80,
```

Comment:
Tested on iOS iPad and the line was clipping out of the card. Going to push a small update to this.